client/signup/navigation-link/test/index.jsx

```diff
@@ -87,24 +87,19 @@ describe( 'NavigationLink', () => {
 	} );
 
 	test( 'should set a proper url as href prop when the direction is "back".', () => {
-		expect( signupUtils.getStepUrl ).not.to.be.called;
+		expect( getStepUrl ).not.to.be.called;
 
 		const wrapper = shallow( <NavigationLink { ...props } direction="back" /> );
 
-		// It should call signupUtils.getStepUrl()
-		expect( signupUtils.getStepUrl ).to.has.been.called;
-		expect( signupUtils.getStepUrl ).to.has.been.calledWith(
-			'test:flow',
-			'test:step1',
-			'test:section1',
-			'en'
-		);
+		// It should call getStepUrl()
+		expect( getStepUrl ).to.has.been.called;
+		expect( getStepUrl ).to.has.been.calledWith( 'test:flow', 'test:step1', 'test:section1', 'en' );
 
 		// when it is the first step
-		signupUtils.getStepUrl = stub();
+		getStepUrl = stub();
```

Comment:
There is a test failure when trying to assign to the constant imported identifier here.

Reply:
Thanks for the catch! I updated the tests in 243e040395 and ended up refactoring them to swap out `chai` and `sinon` with `Jest`